source/guides/puppetlabs_package_verification.markdown

```diff
@@ -35,31 +35,32 @@ Before you can verify signatures, you must import the Puppet Labs public key and
 
 To import the release signing key, run:
```

Comment:
I've been trying to update documentation to refer to the private key as the signing key, and the public key as "public key" lately. </nitpick>